CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.6.1
+
+- Add field `baker_id` in metadata of `BlockResponse`.
```

Comment:
Should there be any changes in the readme, in examples?

Reply:
No it only has examples for the construction endpoints.